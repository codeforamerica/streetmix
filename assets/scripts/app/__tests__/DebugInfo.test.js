/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import { DebugInfo } from '../DebugInfo'

describe('DebugInfo', () => {
  it('render correctly', () => {
    const settings = { }
    const street = { }
    const flags = { }
    const undo = { }
    const wrapper = shallow(<DebugInfo settings={settings} street={street} flags={flags} undo={undo} />)
    expect(wrapper).toMatchSnapshot()
  })
  it('is visible when state visible is set', () => {
    const settings = { }
    const street = { }
    const flags = { }
    const undo = { }
    const wrapper = shallow(<DebugInfo settings={settings} street={street} flags={flags} undo={undo} />)
    wrapper.setState({ visible: true })
    expect(wrapper).toMatchSnapshot()
  })
})
