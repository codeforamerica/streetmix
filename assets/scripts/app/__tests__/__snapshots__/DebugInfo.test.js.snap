// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebugInfo is visible when state visible is set 1`] = `
<div
  className="debug-info visible"
>
  <textarea
    readOnly={true}
    value=""
  />
</div>
`;

exports[`DebugInfo render correctly 1`] = `
<div
  className="debug-info"
>
  <textarea
    readOnly={true}
    value=""
  />
</div>
`;
